pb-crm-api/src/app.module.ts

```diff
@@ -2,38 +2,48 @@ import {Module} from '@nestjs/common';
 import {AuthnModule} from './authn/authn.module';
 import {TypeOrmModule} from "@nestjs/typeorm";
 import {ContactModule} from "./contact/contact.module";
-import { UsersModule } from './authn/users/users.module';
-import {ConfigModule} from "@nestjs/config";
-import { UploadController } from './upload/upload.controller';
-import { UploadService } from './upload/upload.service';
-import { UploadModule } from './upload/upload.module';
-import { MulterModule } from '@nestjs/platform-express';
-
+import {UsersModule} from './authn/users/users.module';
+import {ConfigModule, ConfigService} from "@nestjs/config";
+import {UploadModule} from './upload/upload.module';
+import {MulterModule} from '@nestjs/platform-express';
 import {ErrorService} from "./core/services/error.service";
 
+
+
 @Module({
     imports: [
-        ConfigModule.forRoot(),
         MulterModule.registerAsync({
-          useFactory: () => ({
-            dest: './files'
-          }),
+            useFactory: () => ({
+                dest: './files'
+            }),
+        }),
+        TypeOrmModule.forRootAsync({
+            imports: [ConfigModule.forRoot({
+                envFilePath: `./config/.env.${process.env.ENV}`
+            })],
+            useFactory: (configService: ConfigService) => ({
+                type: 'postgres',
+                host: configService.get('DB_HOST'),
+                port: +configService.get<number>('DB_PORT'),
+                username: configService.get('DB_USR'),
+                password: configService.get('DB_PWD'),
+                database: configService.get('DB_NAME'),
+                entities: ["dist/db/entities/**/*.entity{.js, .ts}"],
+                synchronize: true,
```

Comment:
This should be pulled from config too, we don't want to do this in production or the dev environment. Only on local